src/main/webapp/styles/appointment.css

```diff
@@ -0,0 +1,185 @@
+#appointments {
```

Comment:
I see this file is unused! Could we please delete this file and all commented out references to it.